### DISCUSSION TITLE
Is Search & Replace supported in tiptap 2 on vue 3?

### DISCUSSION THREAD

**Discussion by andon94** [ueberdosis/tiptap#1627] [General Questions]
Can I get some directions if it is?

**Comment by hanspagel** [comment]
Not yet! You can try to port the extensions from the v1 branch.

**Reply by andon94** [reply]
Can I get some more guidance on how to do that please?

**Reply by hanspagel** [reply]
I am afraid I am not able to give you guidance without working on it.
